IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TicketVendingMachine/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of ticket vending machine. The property set can be used by _IfcElectricAppliance_ with PredefinedType _VENDINGMACHINE_.
```

Comment:
Not sure how much consistency is desired, Usually italic was not used either for the entity name nor for the predefined type.
```suggestion
Properties of ticket vending machine. The property set can be used by _IfcElectricAppliance_ with PredefinedType VENDINGMACHINE.
```